src/main/resources/external-policies-api.yaml

```diff
@@ -104,5 +271,299 @@ definitions:
+      subject:
+        type: object
+        title: Subject
+        description: 'Specifies the subject conditions to which the policy applies, where subjects can be combined by using the built-in types "AND", "OR", and "NOT", and where subject implementations are pluggable'
+      condition:
+        type: object
+        properties:
+          type:
+            type: string
+          conditions:
+            type: array
+            items:
+              type: object
+            title: Condition
+            description: 'Specifies environment conditions, where conditions can be combined by using the built-in types "AND", "OR", and "NOT", and where condition implementations are pluggable'
+        title: Condition
+        description: 'Specifies environment conditions, where conditions can be combined by using the built-in types "AND", "OR", and "NOT", and where condition implementations are pluggable'
+      resourceTypeUuid:
+        type: string
+        title: Resource Type UUID
+        description: The UUIDs of the resource type associated with the policy
+      resourceAttributes:
+        type: array
+        items:
+          type: object
+        title: Resource Attributes
+        description: List of attributes to return with decisions. These attributes are known as response attributes
+      lastModifiedBy:
+        type: string
+        title: Last Modified By
+        description: A string containing the universal identifier DN of the subject that most recently updated the policy
+      lastModifiedDate:
+        type: string
+        title: Last Modified date
+        description: 'An integer containing the last modified date and time, in number of seconds'
+      createdBy:
+        type: string
+        title: Created By
+        description: A string containing the universal identifier DN of the subject that created the policy
+      creationDate:
+        type: string
+        title: Creation Date
+        description: 'An integer containing the creation date and time, in number of seconds'
+  PutPolicyResponse:
+    title: Policy Resource Schema
+    description: Json schema for the policy resource
+    type: object
+    properties:
+      name:
+        type: string
+        title: Name
+        description: String matching the name of the application
+      active:
+        type: boolean
+        title: Active flag
+        description: 'Boolean indicating whether OpenAM considers the policy active for evaluation purposes, defaults to false'
+      description:
+        type: string
+        title: Description
+        description: String describing the policy
+      applicationName:
+        type: string
+        title: Application name
+        description: 'String containing the application name, such as "iPlanetAMWebAgentService", or "mypolicyset"'
```

Comment:
Not sure about using an AM example service - I think we try to not be too obvious about the backend